tests/python/pants_test/engine/test_engine.py

```diff
@@ -312,13 +315,26 @@ def test_non_existing_root_fails_differently(self):
 
     @dataclass
     class WorkunitTracker:
-        workunits: List[dict] = field(default_factory=list)
+        """This class records every non-empty batch of started and completed workunits.
+
+        that the engine passes to it, saving them - in the order received from
+        the engine - in the _chunks members.
```

Comment:
Docformatter split these up into two distinct sections because of the period after `completed workunits`. Remove that period, delete the blank line, then run `./v2 fmt` on this file and docformatter should merge this back into one sentence.